aten/src/ATen/native/UnaryOps.cpp

```diff
@@ -543,6 +543,8 @@ Tensor& fix_out(const Tensor& self, Tensor& result) { return at::trunc_out(resul
 Tensor fix(const Tensor& self) { return self.trunc(); }
 Tensor& fix_(Tensor& self) { return self.trunc_(); }
 
+Tensor positive(const Tensor& self) { return at::clone(self); }
```

Comment:
Had a brief offline discussion with @rgommers about returning self, a view, or a copy here and what the new array api required. We think we can just do `return self;` here, because the array api doesn't require copies. What do you think, @asi1024?

Reply:
Yes, I agree to return `self` immediately here :+1:
By the way, now I think that I should have implemented to raise an error when a boolean input is given. How do you think about it?

Reply:
Raising an error for boolean types is consistent with NumPy and I believe that's the intent of the new array API, too, so that sounds good, @asi1024